src/pages/basic-management/employee/list/Index.js

```diff
@@ -0,0 +1,473 @@
+                    value: 1,
+                },
+            ],
+            render(val) {
+                return <Badge status={statusMap[val]} text={status[val]} />;
+            },
+        },
+        {
+            title: '录入人',
+            dataIndex: 'addBy',
+        },
+        {
+            title: '录入日期',
+            dataIndex: 'addOn',
+        },
+        {
+            title: '修改人',
+            dataIndex: 'editBy'
+        },
+        {
+            title: '修改日期',
+            dataIndex: 'editOn',
+        },
+        {
+            title: '操作',
+            fixed: 'right',
+            render: (text, record) => (
+                <Fragment>
+                    <a onClick={() => this.handleEdit(record)}>编辑</a>
+                    <Divider type="vertical" />
+                    <Popconfirm
+                        title="你确定要删除此记录吗?"
+                        onConfirm={() => this.handleDelete(record)}
+                        okText="确定"
+                        cancelText="取消"
+                    >
+                        <a>删除</a>
+                    </Popconfirm>
+                </Fragment>
+            ),
+        },
+    ];
+
+    getColumnSearchProps = dataIndex => ({
+        filterDropdown: ({ setSelectedKeys, selectedKeys, confirm, clearFilters }) => (
+            <div style={{ padding: 8 }}>
+                <Input
+                    ref={node => {
+                        this.searchInput = node;
+                    }}
+                    placeholder={`Search ${dataIndex}`}
+                    value={selectedKeys[0]}
+                    onChange={e => setSelectedKeys(e.target.value ? [e.target.value] : [])}
+                    onPressEnter={() => this.handleSearch(selectedKeys, confirm)}
+                    style={{ width: 188, marginBottom: 8, display: 'block' }}
+                />
+                <Button
+                    type="primary"
+                    onClick={() => this.handleSearch(selectedKeys, confirm)}
+                    icon="search"
+                    size="small"
+                    style={{ width: 90, marginRight: 8 }}
+                >
+                    Search
+        </Button>
+                <Button onClick={() => this.handleReset(clearFilters)} size="small" style={{ width: 90 }}>
+                    Reset
+        </Button>
+            </div>
+        ),
+        filterIcon: filtered => (
+            <Icon type="search" style={{ color: filtered ? '#1890ff' : undefined }} />
+        ),
+        onFilter: (value, record) =>
+            record[dataIndex]
+                .toString()
+                .toLowerCase()
+                .includes(value.toLowerCase()),
+        onFilterDropdownVisibleChange: visible => {
+            if (visible) {
+                setTimeout(() => this.searchInput.select());
+            }
+        },
+        render: text => (
+            <Highlighter
+                highlightStyle={{ backgroundColor: '#ffc069', padding: 0 }}
```

Comment:
Codacy found an issue: [Expected indentation of 14 space characters but found 16.](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=6006324)